docs/content/guides/rows/row-sorting.md

```diff
@@ -83,93 +145,502 @@ import 'handsontable/dist/handsontable.full.min.css';
+        price: 314,
+        sellDate: '27/03/2020',
+        sellTime: '51-6775',
+        inStock: true,
+      },
+      {
+        brand: 'Buzzdog',
+        model: 'Mountain Frame',
+        price: 268,
+        sellDate: '29/08/2020',
+        sellTime: '44-4028',
+        inStock: false,
+      },
+      {
+        brand: 'Katz',
+        model: 'Half-Finger Gloves',
+        price: 204,
+        sellDate: '02/10/2020',
+        sellTime: '08-2758',
+        inStock: true,
+      },
+    ]}
+      columns={[
+        {
+          type: 'text',
+          data: 'brand',
+        },
+        {
+          type: 'text',
+          data: 'model',
+        },
+        {
+          type: 'numeric',
+          data: 'price',
+          numericFormat: {
+            pattern: '$ 0,0.00',
+            culture: 'en-US'
+          },
+        },
+        {
+          type: 'date',
+          data: 'sellDate',
+        },
+        {
+          type: 'text',
+          data: 'sellTime',
+        },
+        {
+          type: 'checkbox',
+          data: 'inStock',
+        },
       ]}
-      colHeaders={['Brand', 'Model', 'Code name', 'Country of origin', 'Rank']}
-      columnSorting={true}
+      colHeaders={[
+        'Brand',
+        'Model',
+        'Price',
+        'Date',
+        'Time',
+        'In stock',
+      ]}
+      columnSorting={{
+        // at initialization, sort rows by column 1, in descending order
+        initialConfig: {
+          column: 1,
+          sortOrder: 'desc',
+        },
+      }}
+      rowHeaders={true}
       height="auto"
+      width="auto"
       licenseKey="non-commercial-and-evaluation"
     />
   );
 };
 
 /* start:skip-in-preview */
-ReactDOM.render(<ExampleComponent />, document.getElementById('example1'));
+ReactDOM.render(<YourHandsontableComponent />, document.getElementById('example1'));
 /* end:skip-in-preview */
 ```
 :::
+
 :::
 
+Only the table view is sorted. Your source data remains in the original order.
+
+## Enable rows sorting
+
+To enable rows sorting, use the [`columnSorting`](@/api/options.md#columnsorting) option.
 
-## Extended plugin configuration
+::: only-for javascript
 
-Plugin options can be customized by providing a configuration as an `object`.
+<code-group>
+  <code-block title="Entire grid">
+  
+  ```js
+  const configurationOptions = {
+    // enable rows sorting for the entire grid
+    columnSorting: true,
+  };
+  ```
+  
+  </code-block>
+  <code-block title="Columns">
+  
+  ```js
+  const configurationOptions = {
+    columns: [
+      {
+        // enable sorting by column 1
+        columnSorting: true,
+      },
+      {
+        // disable sorting by column 2
+        columnSorting: false,
+      },
+    ],
+  };
+  ```
+  
+  </code-block>
+  <code-block title="Rows">
+  
+  ```js
+  const configurationOptions = {
+    // enable rows sorting for the entire grid
+    columnSorting: true,
+
+    // exclude rows 1 and 2 from sorting
+    afterColumnSort() {
```

Comment:
This is great. However, the typical usage is when someone wants to exclude all frozen rows from the sorted range. Can we create an additional example in which we show to exclude those - without passing an exact number of frozen rows?

Reply:
Done